huaweicloud/services/mrs/data_source_huaweicloud_mapreduce_clusters.go

```diff
@@ -0,0 +1,681 @@
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster status.`,
+			},
+			"billing_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster billing mode.`,
+			},
+			"vpc_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `VPC ID.`,
+			},
+			"subnet_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Subnet ID.`,
+			},
+			"duration": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster subscription duration.`,
+			},
+			"fee": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster creation fee, which is automatically calculated.`,
+			},
+			"hadoop_version": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Hadoop version.`,
+			},
+			"master_node_size": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Instance specifications of a Master node.`,
+			},
+			"core_node_size": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Instance specifications of a Core node.`,
+			},
+			"component_list": {
+				Type:        schema.TypeList,
+				Elem:        mrsClustersClustersComponentListSchema(),
+				Computed:    true,
+				Description: `Component list.`,
+			},
+			"external_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `External IP address.`,
+			},
+			"external_alternate_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Backup external IP address.`,
+			},
+			"internal_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Internal IP address.`,
+			},
+			"deployment_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster deployment ID.`,
+			},
+			"remark": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster remarks.`,
+			},
+			"order_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Cluster creation order ID.`,
+			},
+			"master_node_product_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Product ID of a Master node.`,
+			},
+			"master_node_spec_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Specification ID of a Master node.`,
+			},
+			"core_node_product_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Product ID of a Core node.`,
+			},
+			"core_node_spec_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Specification ID of a Core node.`,
+			},
+			"availability_zone": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The AZ.`,
+			},
+			"instance_id": {
```

Comment:
what's the difference between `instance_id` and `id`?

Reply:
the values are the same, delete the instance_id